File: public/docs/_examples/architecture/ts/app/logger.service.ts
// #docregion
import {Injectable} from 'angular2/core';

@Injectable()
// #docregion class
export class Logger {
  log(msg: any)   { console.log(msg); }
  error(msg: any) { console.error(msg); }
  warn(msg: any)  { console.warn(msg); }
}
// #enddocregion class

File: public/docs/_examples/architecture/dart/lib/logger_service.dart
import 'dart:html';

import 'package:angular2/core.dart';

@Injectable()
// #docregion class
class Logger {
  void log(Object msg) => window.console.log(msg);
  void error(Object msg) => window.console.error(msg);
  void warn(Object msg) => window.console.warn(msg);
}
